cmd/collector/main.go

```diff
@@ -46,7 +46,7 @@ func main() {
 	influxClient := db.NewInfluxDBSource(influxOptions)
 	pgClient, err := db.NewPostgresClient(postgresOptions)
 	if err != nil {
-		fmt.Errorf("Error setting up Postgres client: %v\n", err)
+		fmt.Printf("Error setting up Postgres client: %v\n", err)
```

Comment:
This was to conform with a linter error that I was getting in my editor.

Side note: we currently have `golang-ci-lint` disabled in both `pre-commit` and the CI as this repo has a whole ton of linter issues that need to be fixed to conform to the rest of the backend repos' linter style. Once this E2E PR is finished and merged I suggest we turn those linters back on and make all the changes.

Reply:
Sure, I agree we should handle any reasonable suggestions by a linter in a separate PR. Could you revert this change please? Then, as you sugested above, after merging this PR we can handle the linter issues.

Reply:
Unless I'm misunderstanding it, doesn't `fmt.Errorf` merely return a string and not log it? So in this case the existing code wasn't doing anything since the return value was never checked or used.

Reply:
The linter error is `result of fmt.Errorf call not used`

Reply:
I see. My mistake, somehow I read it as `t.Errorf`.